src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -656,3 +658,28 @@ message LongRunningConfigProto {
   // Total polling timeout.
   uint64 total_poll_timeout_millis = 8;
 }
+
+message OutputSpec {
+  message LoopStmt {
+    string collection = 1;
+    string var = 2;
```

Comment:
`var` is a reserved word in JavaScript; it is probably unwise to use it as a field name.

Reply:
Also, we use `string variable` later in this file. :-)

Reply:
Good point.

Using the same term both here and in `Accessor` makes sense, since they both reference a local variable. (And using it eventually in the `AliasStatement` as well for the same reason.) I'm fine with the term `variable`.

Reply:
Done